app/locale/id.coffee

```diff
@@ -1,108 +1,108 @@
 module.exports = nativeDescription: "Bahasa Indonesia", englishDescription: "Indonesian", translation:
   home:
-#    slogan: "Learn to Code by Playing a Game"
-    no_ie: "Maaf, CodeCombat tidak bisa dijalankan pada Internet Explorer 8 dan sebelumnya" # Warning that only shows up in IE8 and older
+#    slogan: "Belajar Membuat Kode Dengan Bermain Sebuah Permainan"
```

Comment:
For example, in this string you would remove the # symbol from the beginning of the sentence:

from
`# slogan: "Belajar Membuat Kode Dengan Bermain Sebuah Permainan"`

to
`slogan: "Belajar Membuat Kode Dengan Bermain Sebuah Permainan"`